sound/soc/sof/intel/hda-dsp.c

```diff
@@ -630,8 +630,8 @@ static int hda_suspend(struct snd_sof_dev *sdev, bool runtime_suspend)
 	snd_hdac_ext_bus_link_power_down_all(bus);
 #endif
 
-	/* power down DSP */
-	ret = snd_sof_dsp_core_power_down(sdev, chip->host_managed_cores_mask);
+	/* power down all DSP cores */
+	ret = snd_sof_dsp_core_power_down(sdev, GENMASK(chip->cores_num - 1, 0));
```

Comment:
the patch looks good. But why is it a fixup? I think the patch is already upstream isnt it? Did you mean to add a Fixes tag instead for this patch?
commit 5607b18eadaca00f8177ec7b9e50cbbe765883b0
Author: Bard Liao <yung-chuan.liao@linux.intel.com>
Date:   Mon Oct 19 13:27:33 2020 +0800

    ASoC: SOF: Intel: hda: use snd_sof_dsp_core_power_up/down API

Reply:
> the patch looks good. But why is it a fixup? I think the patch is already upstream isnt it? Did you mean to add a Fixes tag instead for this patch?
> commit [5607b18](https://github.com/thesofproject/linux/commit/5607b18eadaca00f8177ec7b9e50cbbe765883b0)
> Author: Bard Liao [yung-chuan.liao@linux.intel.com](mailto:yung-chuan.liao@linux.intel.com)
> Date: Mon Oct 19 13:27:33 2020 +0800
> 
> ```
> ASoC: SOF: Intel: hda: use snd_sof_dsp_core_power_up/down API
> ```

Oh, thanks, didn't realize that, let me change.

Reply:
don't bother @keyonjie this PR isn't going to be merged as is, it's fundamentally wrong on multiple levels.